modMovement.bas

```diff
@@ -129,7 +129,7 @@ Dim A As Long, B As Long, C As Long, D As Long
 End Sub
 
 Public Sub MoveProjectiles()
-Dim A As Long, B As Long, C As Long, D As Long
+Dim A As Long, B As Long, C As Long, D As Long, H As Double, X As Long, Y As Long, NewX As Long, NewY As Long

```

Comment:
What is H?
At least for new variables, let's try to use useful names.

Reply:
H was short for Hypotenuse. However these dimensions were eventually unused and therefore can be removed which I will also do...